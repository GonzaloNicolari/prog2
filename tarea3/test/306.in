# prueba de linealizacion
linealizacion
insertar_en_binario (1,ab)
insertar_en_binario (100,abc)
insertar_en_binario (10,aba)
insertar_en_binario (1,abb)
linealizacion
insertar_en_binario (1,abaa)
insertar_en_binario (1,aaaaa)
insertar_en_binario (1,aaaab)
insertar_en_binario (1,aaaaz)
insertar_en_binario (1,aaaza)
linealizacion
Fin
