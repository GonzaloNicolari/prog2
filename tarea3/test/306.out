1>#  prueba de linealizacion.
2>
3>Insertado.
4>Insertado.
5>Insertado.
6>Insertado.
7>(1,ab)(10,aba)(1,abb)(100,abc)
8>Insertado.
9>Insertado.
10>Insertado.
11>Insertado.
12>Insertado.
13>(1,aaaaa)(1,aaaab)(1,aaaaz)(1,aaaza)(1,ab)(10,aba)(1,abaa)(1,abb)(100,abc)
14>Fin.
